src/Tizen.NET.MaterialComponents/Components/MTextField.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Tizen.NET.MaterialComponents
 {
-    public class MTextField : Entry
+    public class MTextField : Entry, IColorSchemeComponent
```

Comment:
If want to apply color scheme system on your widget, need to implement IColorSchemeComponent interface